dbt/adapters/spark/connections.py

```diff
@@ -155,21 +217,22 @@ def open(cls, connection):
         exc = None
         for i in range(1 + connect_retries):
             try:
+                creds = connection.credentials
                 if connection.credentials['method'] == 'http':
```

Comment:
Style note: I would access these things (`method`, `token`, etc) as attributes rather than as dict lookups if you can. It'll make life easier in the future!